src/flask_cognito_lib/decorators.py

```diff
@@ -32,6 +35,63 @@ def remove_from_session(keys: Iterable[str]):
                 session.pop(key)
 
 
+def store_tokens(tokens: CognitoTokenResponse, nonce: Optional[str] = None) -> None:
+    """Store the tokens in the session"""
+    # validate the JWT and get the claims
+    claims = cognito_auth.verify_access_token(
+        token=tokens.access_token,
+        leeway=cfg.cognito_expiration_leeway,
+    )
+    session.update({"claims": claims})
+
+    # Grab the user info from the user endpoint and store in the session
+    if tokens.id_token is not None:
+        user_info = cognito_auth.verify_id_token(
+            token=tokens.id_token,
+            nonce=nonce,
+            leeway=cfg.cognito_expiration_leeway,
+        )
+        session.update({"user_info": user_info})
+
+
+def store_token_in_cookie(
+    token: Union[str, None],
+    cookie_name: str,
+    max_age: int,
+    resp: Response,
+    encrypt: bool = False,
+) -> None:
```

Comment:
Should token always be a string (i.e. required)?
Does it make sense that this function would be called with `token=None`?

Would probably reorder args to put the mandatory ones first:

```suggestion
def store_token_in_cookie(
    resp: Response,
    token: str,
    cookie_name: str,
    max_age: int,
    encrypt: bool = False,
) -> None:
```

Reply:
Reordering done.

> Should token always be a string (i.e. required)?
> Does it make sense that this function would be called with token=None?

In my opinion that doesn't make a lot of sense and would require additional handling of `None` in `TokenService::encrypt_token`